serde_v8/magic/v8slice.rs

```diff
@@ -83,6 +83,16 @@ impl V8Slice {
       std::slice::from_raw_parts_mut(ptr, self.range.len())
     }
   }
+
+  /// Create a [`Vec<u8>`] copy of this slice data.
+  pub fn to_vec(&self) -> Vec<u8> {
+    self.as_slice().to_vec()
+  }
+
+  /// Create a [`Box<[u8]>`] copy of this slice data.
+  pub fn to_boxed_slice(&self) -> Box<[u8]> {
+    self.to_vec().into_boxed_slice()
+  }
```

Comment:
Are these only temporary?

Reply:
Yeah, I think these will go away once we have a better understanding of what async buffers look like and potentially can create a new type. For now, these match the methods available on slices and Vec (both make copies) which should avoid confusion.